pages/tool_assembly/galaxy_assembly.md

```diff
@@ -0,0 +1,220 @@
+        <h3 class="card-title text-center mb-4 ff-theme mt-0 text-white">Collect</h3>
+      </div>
+      <div class="card-body">
+        <h4 class="mt-0">Access to databases</h4>
+        <ul class="lh-sm">
+          <li><a href="https://www.uniprot.org/">UniProt</a></li>
+          <li><a href="http://intermine.org/">InterMine</a></li>
+          <li><a href="https://www.openmicroscopy.org/omero/">OMERO</a></li>
+          <li><a href="https://www.omicsdi.org/">OmicsDI</a></li>
+          <li><a href="https://www.copernicus.eu/en">Copernicus</a></li>
+          <li><a href="https://genome.ucsc.edu/">UCSC genome browser</a> (<a href="https://training.galaxyproject.org/training-material/topics/introduction/tutorials/galaxy-intro-strands/tutorial.html">tutorial</a>)</li>
+          <li><a href="https://www.ncbi.nlm.nih.gov/datasets/">NCBI datasets</a></li>
+          <li><a href="https://www.insdc.org/">INSDC</a> (<a href="https://www.ebi.ac.uk/ena/browser/about">ENA</a>)</li>
+          <li><a href="https://www.rcsb.org/">PDB</a></li>
+          <li>3rd-party databases</li>
+        </ul>
+        <h4>Customised data access</h4>
+        <ul class="lh-sm">
+          <li>Data libraries</li>
+          <li>BYOD (Posix, WebDav, Dropbox, ...)</li>
+          <li>On-demand reference data</li>
+          <li><i>Deferred</i> data from remote locations</li>
+        </ul>
+        <h4>LIMS integration</h4>
+        <ul class="lh-sm">
+          <li>Connect to sequencing facilities</li>
+          <li>Rich API for integration with LIMS</li>
+        </ul>
+      </div>
+    </div>
+  </div>
+  <div class="col">
+    <div class="card border-2 h-100" style="border-color: #8aba56!important">
+      <div style="background-color:#8aba56;">
+        <img src="{{ '/images/life_cycle_icons/noun-computer.svg' | relative_url }}" class="card-img-top pt-3" style="max-height: 6em;" alt="process">
+        <h3 class="card-title text-center mb-4 ff-theme mt-0 text-white">Process</h3>
+      </div>
+      <div class="card-body">
+        <h4 class="mt-0">Data transformation</h4>
+        <ul class="lh-sm">
+          <li>Data transformation tools</li>
+          <li>Quality control</li>
+          <li>Data cleaning</li>
+          <li>Annotation</li>
+          <li><a href="https://live.usegalaxy.eu/">Interactive Tools</a> (<a href="https://openrefine.org/">OpenRefine</a>, RStudio, Jupyter Notebook)</li>
+        </ul>
+        <h4>Import workflows</h4>
+        <ul class="lh-sm">
+          <li><a href="https://workflowhub.eu/">WorkflowHub</a></li>
+          <li><a href="https://dockstore.org/">Dockstore</a></li>
+          <li><a href="https://www.ga4gh.org/news/tool-registry-service-api-enabling-an-interoperable-library-of-genomics-analysis-tools/">GA4GH TRS API</a></li>
+        </ul>
+        <h4>Metadata handling</h4>
+        <ul class="lh-sm">
+          <li>Provenance tracking</li>
+          <li>Automatic metadata enrichment</li>
```

Comment:
I feel this could be more extensive

Reply:
@beatrizserrano any comments?

Reply:
Not much from my side, I think. I don't know if there's something generic enough to be specified here, what do you think @bgruening?

Reply:
@korbinib if @bgruening gives green light, and you give your approval, you can merge. I am on holiday tomorrow :)

Reply:
We could add here more (like datatypes, EDAM annotaitons, number of columns, reference genome ...), the question is just if it's worth it and how. I guess for the first PR this is good and we keep adding things to those list later. :+1: from my side. Lets get it in :)